### DISCUSSION TITLE
Windows: Error when trying to Build Erigon

### DISCUSSION THREAD

**Discussion by itgav** [erigontech/erigon#4687] [Q&A] [answered]
I'm trying to download Erigon for the first time and keep getting an error when attempting to build. I'm a noob can someone PLEASE give me some guidance on what might be the issue. I've been trying to find threads that might help me solve the problem but have been unsuccessful.

System Info:
Windows 11 OS 64 bit
Go version: 1.18
MinGW version: 11.2

![Erigon Error](https://user-images.githubusercontent.com/94311667/178122911-f31762c2-1f05-4f47-aacf-739900b699d8.png)

**Comment by itgav** [comment]
There was an error in the eth_receipts.go file, it is fixed now. There was a function with a parameter "Block" that didn't have a proper reference.

**Comment by AskAlexSharov** [comment, marked as answer]
Try “git checkout alpha”

**Reply by itgav** [reply]
Will do, thank you Alex!
